WcaOnRails/config/locales/en.yml

```diff
@@ -365,6 +365,18 @@ en:
         guests_enabled:
           "true": "Ask about guests"
           "false": "Do NOT ask about guests"
+  mail_form:
```

Comment:
I'm a bit curious about this: how does `i18n-tasks` know about them?!
It doesn't seem to report any unused key for the English locale, yet `i18n-tasks find "mail_form.attributes.*"` returns nothing!

I would have expected the custom scanner to be extended to take these into account, but it looks like it's not necessary.
Do you have any about what's going on?